sql/2024/seo/html-response-vary-header-used-2024.sql

```diff
@@ -0,0 +1,30 @@
+WITH subquery AS (
+  SELECT
+    client,
+    request_headers,
+    CASE
+      WHEN is_root_page = FALSE THEN 'Secondarypage'
+      WHEN is_root_page = TRUE THEN 'Homepage'
+      ELSE 'No Assigned Page'
+    END AS is_root_page
+  FROM
+    `httparchive.all.requests`
+  WHERE
+    date = '2024-06-01'
+)
+
+SELECT
+  client,
+  is_root_page,
+  REGEXP_CONTAINS(LOWER(IFNULL(request_headers[SAFE_OFFSET(0)].name, '')), r'user-agent') AS resp_vary_user_agent,
+  COUNT(0) AS freq,
+  SAFE_DIVIDE(COUNT(0), SUM(COUNT(0)) OVER ()) AS pct
```

Comment:
Ditto

Reply:
This metric was not utilised in the end, easier to manipulate within a pivot table. I have removed from each file